README.md

```diff
@@ -525,3 +525,65 @@ will output
   </div>
 </div>
 ```
+
+### Button
+
+An accessible way to add button links into content, that can also be tracked.
```

Comment:
Worth explicitly specifying here that it can be tracked using Google Analytics?

Reply:
Definitely sounds like a good idea, will update